app/controllers/jobs_controller.rb

```diff
@@ -41,8 +29,8 @@ def index
     end
 
     # Get the complete lists of live and dead jobs from the server
-    raw_live_jobs = getCurrentJobs
-    raw_dead_jobs = getDeadJobs
+    raw_live_jobs = TangoClient.tango_jobs()["jobs"]
```

Comment:
Tango returns a full json, with not only `jobs` but also `statusId`. Although we don't use it here, I thought it might be good to just have the full json and do whatever with it, just like the other endpoints.